x-pack/solutions/security/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -500,9 +460,9 @@ const StatefulEventsViewerComponent: React.FC<EventsViewerProps & PropsFromRedux
     tableId,
     data: nonDeletedEvents,
     totalItems: totalCountMinusDeleted,
-    hasAlertsCrud: hasCrudPermissions,
+    hasAlertsCrud: true,
```

Comment:
Same here

Reply:
see my answer https://github.com/elastic/kibana/pull/206523#discussion_r1915428592 and let me know if you're ok with it!